src/browser/atom-window.coffee

```diff
@@ -64,23 +62,18 @@ class AtomWindow
 
     loadSettings.initialPaths.sort()
 
-    @browserWindow.loadSettings = loadSettings
     @browserWindow.once 'window:loaded', =>
       @emit 'window:loaded'
       @loaded = true
 
     @setLoadSettings(loadSettings)
     @browserWindow.focusOnWebView() if @isSpec
+    @browserWindow.temporaryState = {windowDimensions} if windowDimensions?
```

Comment:
Now, instead of making the window dimensions a top-level field on the load settings, we pre-populate the window's 'temporary state' (see below). Since this temporary window state is overwritten periodically and when unloading the window, we should avoid the unwanted cascade positioning.